pkg/broker/storage.go

```diff
@@ -268,7 +269,24 @@ begin
             
-    end if;
+		end if;
+		
+		-- add AWS postgres T3 plans (if not present)
+
+		if (select count(*) from plans where plan = '80bf8ed1-4268-49cc-acf8-148be11eda41') = 0 then
+			insert into plans
+				(plan, service, name, human_name, description, version, type, scheme, categories, cost_cents, preprovision, attributes, provider, provider_private_details)
+			values
+				('80bf8ed1-4268-49cc-acf8-148be11eda41', '01bb60d2-f2bb-64c0-4c8b-ead731a690bd', 'premium-t3-0-v9', 'Premium-T3-0 (9.6)', 'Postgres 9.6.9 - 2xCPU 2GB Ram 20GB Storage (T3 Instance)', '9.5.2', 'postgres', 'postgres', 'Data Stores', 2000,  0, '{"compliance":"", "supports_extensions":true,  "ram":"2GB",   "database_replicas":true,  "database_logs":true,  "restartable":true,  "row_limits":null, "storage_capacity":"20GB",  "data_clips":true,  "connection_limit":null, "high_availability":false,  "rollback":"14 days", "encryption_at_rest":true, "high_speed_ssd":true,  "burstable_performance":false, "dedicated":true  }', 'aws-instance',    '{"AllocatedStorage":20,"AutoMinorVersionUpgrade":true,"AvailabilityZone":null,"BackupRetentionPeriod":14,"CharacterSetName":null,"CopyTagsToSnapshot":true,"DBClusterIdentifier":null,"DBInstanceClass":"db.t3.small","DBInstanceIdentifier":null,"DBName":null,"DBParameterGroupName":null,"DBSecurityGroups":null,"DBSubnetGroupName":"rds-postgres-subnet-group","Domain":null,"DomainIAMRoleName":null,"EnableCloudwatchLogsExports":null,"EnableIAMDatabaseAuthentication":null,"EnablePerformanceInsights":false,"Engine":"postgres","EngineVersion":"9.6.9","Iops":null,"KmsKeyId":null,"LicenseModel":null,"MasterUserPassword":null,"MasterUsername":null,"MonitoringInterval":null,"MonitoringRoleArn":null,"MultiAZ":false,"OptionGroupName":null,"PerformanceInsightsKMSKeyId":null,"PerformanceInsightsRetentionPeriod":null,"Port":null,"PreferredBackupWindow":null,"PreferredMaintenanceWindow":null,"ProcessorFeatures":null,"PromotionTier":null,"PubliclyAccessible":false,"StorageEncrypted":true,"StorageType":"gp2","Tags":null,"TdeCredentialArn":null,"TdeCredentialPassword":null,"Timezone":null,"VpcSecurityGroupIds":null}');
```

Comment:
We've deprecated 9.X plans, Amazon will more than likely remove the ability to provision them soon.  Postgres is now on v12, so we should standardize to v11 for new plans unless there's a strong reason to keep on 9.

Reply:
Sounds good - I'll remove the v9 and bump the other to v11